ruby/files_and_serialization/project_event_manager.md

```diff
@@ -694,7 +697,7 @@ end
 Running our application, we find an error.
 
 ```bash
-$ ruby lib/event_manager.rb
+$ ruby bundle exec ruby lib/event_manager.rb
```

Comment:
For these other commands shown throughout the lesson, the `$` is typically fine because it's meant to show the output for a particular command.

However, some of these (like this one) have the command as `$ ruby bundle exec ruby ...`. This should be changed to `$ bundle exec ruby ...`.

Reply:
Thank you, Josh! command was updated